packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -163,3 +172,43 @@ export const getContractTypes = (list: TList[], item: TItem) => findContractCate
 
 export const getCategoriesSortedByKey = (list: TContractCategory[] = []) =>
     [...list].sort((a, b) => ordered_trade_categories.indexOf(a.key) - ordered_trade_categories.indexOf(b.key));
+
+export const CF_STREAM_CUSTOMER_URL = 'https://customer-hhvo3ceuqt00w8g8.cloudflarestream.com';
```

Comment:
Are we supposed to hardcode all these values here?

Reply:
@akmal-deriv yes, it will be needed only here, and the CF_STREAM_CUSTOMER_URL is public, it's ok to state it in the code. :)

Reply:
What I mean is isn't it better to move URL and video IDs into one separate config and get/update values there in the future?

Reply:
@akmal-deriv i've created a separate file for this - video-config.ts